packages/core/src/components/Header/Actions/Actions.styles.tsx

```diff
@@ -1,19 +1,20 @@
-import styled from "@emotion/styled";
-
 import { theme } from "@hitachivantara/uikit-styles";
 
 import { buttonClasses } from "@core/components/Button";
+import { createClasses } from "@core/utils/classes";
 
-export const StyledDiv = styled.div({
-  backgroundColor: "transparent",
-  display: "flex",
-  gap: theme.space.xs,
-  alignItems: "center",
-  marginLeft: "auto",
-  "& > *": {
-    marginLeft: theme.space.xs,
-  },
-  [`& .${buttonClasses?.root}`]: {
-    "&:hover": { backgroundColor: theme.header.hoverColor },
+export const { staticClasses, useClasses } = createClasses("HvHeader-Actions", {
+  root: {
+    backgroundColor: "transparent",
+    display: "flex",
+    gap: theme.space.xs,
+    alignItems: "center",
+    marginLeft: "auto",
+    "& > *": {
+      marginLeft: theme.space.xs,
+    },
```

Comment:
Unrelated to the refactor, but can we remove this? 😬

```suggestion
```

I noticed that we have duplicated spacing (`gap` + `marginLeft`)

Reply:
I'll take a look at this!